### ISSUE TITLE
Search for a new maintainer?

### ISSUE THREAD

**Issue by arthurvr** [dypsilon/frontend-dev-bookmarks#314]
@dypsilon Don't take me wrong. You've been doing an awesome job. I just think the huge PR count is sad and maybe people are willing to help out.

**Comment by podo** [comment]
Why not having several collaborators?

**Comment by dypsilon** [comment]
Thank you for your suggestions. The reason this project looks stale is that I'm working on a new and improved version of the list every day. This version will incorporate new content and much better usability and solve many "issues" and "PRs" at once.

**Comment by arthurvr** [comment]
@dypsilon If you're planning to change the format of this list drastically, [please don't.](https://github.com/sindresorhus/awesome/pull/209#issuecomment-115785736)

**Comment by dypsilon** [comment]
@arthurvr could you provide some advantages of github readmes over websites? What are the main features of such lists you couldn't find on websites?

The design in your example is just a bad fit for this kind of information. I really want to try and find a better way that will satisfy people who want to work with large github lists and those who want better usability.

**Comment by arthurvr** [comment]
Because everybody can contribute without any extra overhead, because everybody knows how to write markdown, because it's consistent, because GitHub has all infrastructure (issues/PRs) you need, ...

**Comment by dypsilon** [comment]
Great, thank you! I have all those points in mind and the new version will offer the best of both worlds.

**Comment by arthurvr** [comment]
I don't see why you need anything else than GitHub. Why would you want anymore than this?

**Comment by nicovanzyl** [comment]
Why don't you try use [Github pages](https://pages.github.com/)?

It's simple, hosted on the repo and everyone can edit it.

**Comment by dypsilon** [comment]
The Readmes scale really badly:

+ Inconsistencies like lack of a semicolon or lack of a description sneak in.
+ The in-browser search function is only so good.
+ It's hard to keep the resources sorted (alphabetically) and there is only one way to sort them. People are sending pull requests with their contribution on the top of a category.
+ Whenever a new abstraction level is added, it's hard to follow the overall content structure of the list.
+ Contributing becomes harder when there are many resources in the list. It takes time to find the right position.
+ Pull Requests are really bad for the maintainer because there are many conflicts.
+ It's hard to automate maintenance tasks like: finding broken links.

**Comment by arthurvr** [comment]
> Inconsistencies like lack of a semicolon or lack of a description sneak in.

Just review PRs properly and clean it up when you see inconsistencies. Clear contribution guidelines help too.

> The in-browser search function is only so good.

GitHub has a good search feature. No need to re-invent the wheel.

> It's hard to keep the resources sorted (alphabetically) and there is only one way to sort them. People are sending pull requests with their contribution on the top of a category.

Again, just review PRs properly and make sure to have clear contribution guidelines.

> Pull Requests are really bad for the maintainer because there are many conflicts.

Of course if you let every PR wait months ;)

**Comment by dypsilon** [comment]
I think guidelines is another hurdle for contributions. They are only necessary because github is a service for source management, not bookmark management. Many suggestions are only workarounds to make github work with this kind of projects. If you have a hammer...

@arthurvr I would gladly add you as a collaborator for the time being if you keep in mind that a new version is coming.

**Comment by arthurvr** [comment]
No thanks. I'm not really interested mainly because of that new version. IIRC that kind of sites have been a reason to not allow specific lists on https://github.com/sindresorhus/awesome too.

**Comment by dypsilon** [comment]
So what is your suggestion here?

**Comment by arthurvr** [comment]
Maintain the list in a GitHub repo as it is right now. No need for a site IMHO and that only makes it difficult for both contributors and for you in the long-term.

**Comment by dypsilon** [comment]
@arthurvr I hear you loud and clear. I will improve the list to adhere to the quality standards of "awesome".

As for the website: what I had in mind is an export feature which will generate a flat list and post it on GitHub regularly. I think this way everyone will be happy.

**Comment by ngryman** [comment]
@dypsilon For maintenance, I think you could split this huge list in several `md` files organized by themes, the `readme` being a `toc`.

A website could be great imo because it would allow more flexibility to display, illustrate, sort, filter content. However I agree that it should be mirrored some way on Github.

Perhaps you could simply generate your website from your `md` files. This would enforce a clear syntax on how to write each *bookmark*.
